axon-server-connector/pom.xml

```diff
@@ -20,7 +20,7 @@
     <parent>
         <groupId>org.axonframework</groupId>
         <artifactId>axon</artifactId>
-        <version>4.3-SNAPSHOT</version>
+        <version>4.4-SNAPSHOT</version>
```

Comment:
Can we do a patch version number here instead? Like `4.3.1-SNAPSHOT`. Or maybe even `4.3.0.1-SNAPSHOT`. Ideally we don't want to collide with real Axon version numbers or it'll potentially get confusing.

Reply:
ah, yeah, that sounds reasonable!